redaxo/src/core/lib/list.php

```diff
@@ -345,6 +355,60 @@ public function getLinkAttributes($column, $default = null)
         return $this->linkAttributes[$column] ?? $default;
     }
 
+    // row attribute setter/getter etc
+
+    /**
+     * Methode, um dem der Zeile (<tr>) Attribute hinzuzufügen.
+     *
+     * @param mixed $attr   entweder ein array: [attributname => attribut, ...]
+     *                      oder eine Callback-Funktion
+     */
+    public function setRowAttributes($attr)
+    {
+        if (is_callable($attr)) {
+            $this->rowAttributes = $attr;
+            $this->rowAttributesCallable = true;
+        } elseif (is_array($attr) && count($attr)) {
+            $this->rowAttributes = $attr;
+            $this->rowAttributesCallable = false;
+        } else {
+            $this->rowAttributes = [];
+            $this->rowAttributesCallable = false;
+        }
+    }
+
+    /**
+     * Methode, um die aktuellen Attribute der Zeile (<tr>) Attribute abzufragen.
+     *
+     * @return mixed    entweder ein array: [attributname => attribut, ...]
+     *                  oder eine Callback-Funktion
+     */
+    public function getRowAttributes()
+    {
+        return $this->rowAttributes;
+    }
+
+    /**
+     * Methode, um die aktuellen Attribute als Attribut-String auszugeben.
+     *
+     * @return string   attributname="..." ....
+     */
+    protected function fetchRowAttributes()
```

Comment:
```suggestion
    private function fetchRowAttributes(): string
```

Hier möchte ich lieber mit `private` starten, solange niemand ganz konkret es als protected benötigt.

Reply:
für mich ok